src/cmake/modules/clang-format.cmake

```diff
@@ -0,0 +1,4 @@
+find_program(CLANG_FORMAT NAMES clang-format-9)
```

Comment:
Wouldn't just `clang-format` be better? Is there any reason for specifying the version?